.github/workflows/testground-on-push.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Testground PR Checker
+
+on: [push]
+
+jobs:
+  testground:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.composition_file }}
+    strategy:
+      matrix:
+        include:
+          - backend_addr: ci.testground.ipfs.team
+            backend_proto: https
+            plan_directory: testplans/graphsync
+            composition_file: testplans/graphsync/_compositions/stress-k8s.toml
```

Comment:
This one probably doesn't belong to Lotus. I think if we have the graphsync repo first the graphsync tests, it'd be better!

Reply:
Thanks, I'll remove this one.